<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCSwarmControl</class>
 <widget class="QWidget" name="QGCSwarmControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="continueAllButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Return2Start">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to start</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="scenarioLaunch">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set scenario</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="radiusSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="numVhcSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Num. of vehicles</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Circle Radius</string>
   </property>
  </widget>
  <widget class="QPushButton" name="autoLanding">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto landing</string>
   </property>
  </widget>
  <widget class="QListWidget" name="scenarioList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Circle</string>
    </property>
    <property name="checkState">
     <enum>Checked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Circle uniform</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stream</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="autoTakeoff">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto takeoff</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="altitudeSpinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoContinueCheckBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto continue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
